pkg/aws/aws_client/role.go

```diff
@@ -45,13 +46,14 @@ func (client *AWSClient) CreateRole(roleName string,
 	if len(tags) != 0 {
 		input.Tags = roleTags
 	}
-
-	resp, err := client.IamClient.CreateRole(context.TODO(), input)
+	var resp *iam.CreateRoleOutput
+	resp, err = client.IamClient.CreateRole(context.TODO(), input)
 	if err != nil {
-		return *resp.Role, err
+		return
 	}
+	role = *resp.Role
```

Comment:
can we wrap this in a nil check to be sure. We can still hit the nil pointer when we dereference here.

Reply:
Thanks. Updated.